editor/src/core/model/element-metadata-utils.ts

```diff
@@ -315,6 +315,50 @@ export const MetadataUtils = {
     const instance = MetadataUtils.findElementByElementPath(metadata, target)
     return MetadataUtils.isButtonFromMetadata(instance)
   },
+  isTextFromMetadata(element: ElementInstanceMetadata | null): boolean {
+    const elementName = MetadataUtils.getJSXElementName(maybeEitherToMaybe(element?.element))
+    if (
+      elementName != null &&
+      PP.depth(elementName.propertyPath) === 0 &&
+      (elementName.baseVariable === 'p' ||
+        elementName.baseVariable === 'span' ||
+        elementName.baseVariable === 'h1' ||
+        elementName.baseVariable === 'h2' ||
+        elementName.baseVariable === 'h3' ||
+        elementName.baseVariable === 'h4' ||
+        elementName.baseVariable === 'h5' ||
+        elementName.baseVariable === 'h6')
```

Comment:
I'd suggest storing these strings in an array in `dom-utils.ts` like we do with e.g. [`intrinsicHTMLElementNamesThatSupportChildren `](https://github.com/concrete-utopia/utopia/blob/6f8dbc103f8e83dd311a7f6a4166ac91622e9659/editor/src/core/shared/dom-utils.ts#L179), then we could use a `foldEither` on the `element.element` along the lines of:
```ts
if (element == null) { return false }
const isTextElement = foldEither(
  (elementString) => textElements.includes(elementString),
  (elementInstance) => 
    // TS might be alright with this...
    isJSXElement(elementInstance) && 
    textElements.includes(elementInstance.name.baseVariable),
  element.element
)
```
(that's based on the far more complex than we need here [`targetElementSupportsChildrenAlsoText`](https://github.com/concrete-utopia/utopia/blob/6f8dbc103f8e83dd311a7f6a4166ac91622e9659/editor/src/core/model/element-metadata-utils.ts#L732))